exist-core/src/main/java/org/exist/repo/AutoDeploymentTrigger.java

```diff
@@ -77,6 +77,7 @@ public void execute(final DBBroker sysBroker, final Txn transaction, final Map<S
         }
 
         if (!Files.isReadable(autodeployDir) && Files.isDirectory(autodeployDir)) {
+            LOG.warn("Unable to read autodeploy directory: " + autodeployDir);
```

Comment:
The proposed fix for the code smell reported by SonarCloud
`LOG.warn("Unable to read autodeploy directory: {}", autodeployDir)`

Reply:
It is consistent with style of rest of code.